docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -3988,6 +3988,133 @@ In order to use the WIPS feature in PacketFence, please follow those simple step
 First, configure PacketFence to be a trap receiver.  Under *Configuration > SNMP > Trap Receivers*, add an entry for the PF management IP.  By default, all traps will be enabled.  If you want to disable some, you will need
 to connect via CLI, and run the *snmp-server trap disable <trapname>* command.
 
+WebAuth
++++++++
+
+First of all you will need to configure a guest VLAN.
+
+image::docs/images/aruba-config-guest-vlan.png[scaledwidth="100%",alt="Guest VLAN"]
+
+Now you will need to configure a RADIUS server.
+
+image::docs/images/aruba-radius-server.png[scaledwidth="100%",alt="RADIUS server"]
+
+  aaa authentication-server radius "packetfence"
+  host 192.168.1.5
+  key useStrongerSecret
+
+Add your RADIUS server to a AAA group, under `Security > Authentication > Servers > Server Group`:
+
+  aaa server-group "packetfence"
+  auth-server "packetfence" position 1
+
+Then define the `RFC 3576` server, which will allow you to do CoA.
+
+image::docs/images/aruba-config-coa.png[scaledwidth="100%",alt="RADIUS CoA server"]
+
+  aaa rfc-3576-server "192.168.1.5"
+  key useStrongerSecret
+
+Next you will need to create the policy that will redirect user to the PacketFence captive portal while not authenticated. Go to `Security > Authentication > L3 Authenticaion > Captive Portal Authentication Profile`.
```

Comment:
that will redirect users to the PacketFence captive portal when they are not authenticated

Fix 'users' and the structure of the end of the sentence